config/samples/templates/psm/csharp_example_loadtest_proxied.yaml

```diff
@@ -0,0 +1,135 @@
+# Load test configurations generated from a template by loadtest_config.py.
+# See documentation below:
+# https://github.com/grpc/grpc/blob/master/tools/run_tests/performance/README.md#grpc-oss-benchmarks
+apiVersion: e2etest.grpc.io/v1
+kind: LoadTest
+metadata:
+  annotations:
+    scenario: csharp_protobuf_async_unary_5000rpcs_1KB_psm_8channels_16threads_5000offered_load
+    uniquifier: proxied
+  labels:
+    language: csharp
+    prefix: psm-examples
+  name: psm-examples-csharp-protobuf-async-unary-5000rpcs-1kb-psm-8channels-16threads-5000offered-load-proxied
```

Comment:
I think we changed "offered-load" to just "load" in the names, perhaps we have to generate the templates again.